corehq/apps/groups/templates/groups/all_groups.html

```diff
@@ -35,6 +35,16 @@
 {% endblock %}
 
 {% block main_column %}
+{% if needs_to_downgrade_locations %}
+    <div>
+        <p>
+            {% trans "This project no longer has access to the Organization feature." %}
+            <a href="{% url "downgrade_locations" domain %}">
+                {% trans "Click here to fix this." %}
```

Comment:
That's actually how I initially implemented it, but Product preferred this way.  Also, this is expected to be used very rarely, only when a domain uses locations, then downgrades their plan and loses access to locations.